hpeOneView/resources/servers/id_pools.py

```diff
@@ -23,64 +23,84 @@
 
 standard_library.install_aliases()
 
-from hpeOneView.resources.resource import ResourceClient
+from hpeOneView.resources.resource import Resource, ResourceSchemaMixin
 
 
-class IdPools(object):
+class IdPools(Resource, ResourceSchemaMixin):
     """
     Class for Id Pools API client.
     """
     URI = '/rest/id-pools'
 
-    def __init__(self, con):
-        self._client = ResourceClient(con, self.URI)
+    def __init__(self, connection, data=None):
+        super(IdPools, self).__init__(connection, data)
 
-    def get(self, id_or_uri):
+    def get(self):
         """
         Gets a pool.
-
         Args:
             id_or_uri: Can be either the range ID or URI.
-
         Returns:
             dict: Pool resource.
         """
-        return self._client.get(id_or_uri)
+        return self._helper.do_get(self.URI)
```

Comment:
is this working?
because URI needs to be /rest/id-pools/{poolType} and self.URI is /rest/id-pools/

Reply:
get_pool_type is meant for /rest/id-pools/{poolType}(GET) which works fine. First get method is old and not being used. I will remove it.

Reply:
please remove this get method

Reply:
Done